server/src/lsp-server.ts

```diff
@@ -767,11 +776,15 @@ export class LspServer {
             }
         } else if (arg.command === Commands.ORGANIZE_IMPORTS && arg.arguments) {
             const file = arg.arguments[0] as string;
+            const additionalArguments = arg.arguments[1] as undefined | {
+                skipDestructiveCodeActions?: boolean;
+            };
             const { body } = await this.tspClient.request(CommandTypes.OrganizeImports, {
                 scope: {
                     type: 'file',
                     args: { file }
-                }
+                },
+                ...additionalArguments
```

Comment:
I would like to be more "defensive" here and only pass the properties that we expect here. Otherwise this can go bad at runtime.